CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,132 @@
+# Contributor Covenant Code of Conduct
```

Comment:
FYI, we don't need this because it's in this repo: https://github.com/abcxyz/.github. `.github` is a special repo where we can host things like contribution guides, codes of conduct, and issue templates. I would suggest removing from here, since it's inherited from .github (and centrally managed).